tests/src/Traits/TraversingTrait.php

```diff
@@ -350,6 +350,40 @@ protected function findFacetByAlias(string $alias, ?NodeElement $region = NULL,
     return $element;
   }
 
+  /**
+   * Finds a facet form by alias.
+   *
+   * @param string $alias
+   *   The facet form alias.
+   * @param \Behat\Mink\Element\NodeElement|null $region
+   *   (optional) Limit the search to a specific region. If empty, the whole
+   *   page will be used. Defaults to NULL.
+   * @param string $html_tag
+   *   (optional) Limit to a specific html tag when searching for an element.
+   *   This can be useful in cases where the data drupal facet id is placed in
+   *   more than one html tag e.g. the dropdown has the id placed in both the
+   *   <li> tag of links as well as the <select> element.
+   *
+   * @return \Behat\Mink\Element\NodeElement
+   *   The facet node element.
+   *
+   * @throws \Exception
+   *   Thrown when the facet form is not found in the designated area.
+   */
+  protected function findFacetFormByAlias(string $alias, ?NodeElement $region = NULL, string $html_tag = '*'): NodeElement {
+    if ($region === NULL) {
+      $region = $this->getSession()->getPage();
+    }
+    $facet_id = self::getFacetFormIdFromAlias($alias);
```

Comment:
To be honest I don't like the fact that we are duplicating the documentation as well as most of the method in order to look in a different list with a different data-drupal property.
I would prefer to somehow unify them. But again, this is also optional. It won't be too much of an improvement.

Reply:
Actually, there is a good way to improve it. I see that the `getFacetFormIdFromAlias` is used only here, and the `getFacetIdFromAlias` is used only in 3 places. Furthermore, in all cases, the ID is retrieved to generate an xpath with the corresponding data-drupal property.
We can unify the above with a parameter of the type of the facet (e.g. facet or facet-form) and have it return the xpath (including the data-drupal property) directly instead of simply returning the identifier.
I mean, this is just one idea, but we can find ways to unify them.

Reply:
If the ticket is way over board, you can ignore this bit.